src/App.jsx

```diff
@@ -14,70 +16,69 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [dish, setDish] = useState('Jam');
```

Comment:
Would be great to stay strict with the task description
![image](https://github.com/mate-academy/react_goods-selector-js/assets/96409563/585bec2d-a69a-414b-9fb7-e0492e24c5fe)